packages/nimble-components/src/anchor/index.ts

```diff
@@ -34,6 +34,35 @@ export class Anchor extends AnchorBase {
      */
     @attr
     public appearance: AnchorAppearance;
+
+    /**
+     * @internal
+     */
+    public container?: HTMLElement;
```

Comment:
You can just declare this as:
```
public container!: HTMLElement;
```
and thus avoid the unnecessary `!` checks later.